src/biogeophys/WaterDiagnosticBulkType.F90

```diff
@@ -277,8 +286,28 @@ subroutine InitBulkHistory(this, bounds)
          fname=this%info%fname('TOTSOILICE'),  &
          units='kg/m2', &
          avgflag='A', &
-         long_name=this%info%lname('vertically summed soil cie (veg landunits only)'), &
+         long_name=this%info%lname('vertically summed soil ice (veg landunits only)'), &
          ptr_col=this%h2osoi_ice_tot_col, l2g_scale_type='veg')
+    ! excess ice vars
+    if (use_excess_ice) then
+      this%exice_vol_tot_col(begc:endc) = 0.0_r8
+      call hist_addfld1d ( &
+           fname=this%info%fname('TOTEXICE_VOL'),  &
+           units='m3/m3',  &
+           avgflag='A', &
+           l2g_scale_type='veg', &
+           long_name=this%info%lname('vertically averaged volumetric excess ice concentration (veg landunits only)'), &
+           ptr_col=this%exice_vol_tot_col)
```

Comment:
Should both of these output by default anytime excess ice is turned on? They could be set to "inactive" and then turned on in the namelist when they are desired. We output so much that it's good to be careful on the default output we add.

Reply:
At some point, I had them inactive, but then I assumed that maybe it is better to just have an if block there, so users when they run with `use_excess_ice=.true.` don't need to worry to switch the history variables on. I don't know if people run CTSM mostly with standard active output or change it most of the time. The diagnostic variable of course can be calculated in post-processing from the prognostic one but that requires some knowledge of how the parameterization works.

Reply:
This is probably something that would be good to discuss on a call. Eventually excess ice is likely to be a pretty standard way to run the model, so some people will be using it, but not looking into the details of what it's outputting. So I think these probably should be within the use_excess_ice if, but unless they are really standard things that everyone should be outputting, they should also have default='inactive'

Reply:
Let's make this top one inactive by default, and the other two active by default, when excess_ice is on.